RubberduckTests/UnitTesting/MockedTestExplorer.cs

```diff
@@ -14,7 +14,7 @@ public MockedTestExplorer(MockedTestExplorerModel model)
             Vbe = model.Engine.Vbe.Object;
             State = model.Engine.ParserState;
             Model = model.Model;
-            ViewModel = new TestExplorerViewModel(null, Model, ClipboardWriter.Object, null, null);
+            ViewModel = new TestExplorerViewModel(null, Model, ClipboardWriter.Object, null, null, null, null);
```

Comment:
As a general rule, we should avoid newing things in tests but rather reference to a single static method `Arranage...`. That helps us avoid having a ugly refactoring session when we need to modify the parameters for the ctor. This puppy is already up to 7 parameters 😨 so it's surely itching to be refactored.